app/Jasonette/JasonViewController.m

```diff
@@ -1321,8 +1318,9 @@ - (void)setupHeader: (NSDictionary *)body{
                             }
                         }
 
-                        if(!self.tableView.tableHeaderView) {
-                            self.tableView.tableHeaderView = self.searchController.searchBar;
+                        if (@available(iOS 13.0, *)) {
```

Comment:
What is the lowest iOS version for this project?
In my project file it is iOS 13, if so, do we need this `if` statement?

Reply:
no, i will remove the **if** statement.